ApiDoctor.Console/Program.cs

```diff
@@ -2896,6 +2893,41 @@ private static async Task<bool> GeneratePermissionFilesAsync(GeneratePermissionF
             return true;
         }
 
+        private static List<string> PermissionKeywordsToIgnore = new List<string>()
+        {
+            "None",
+            "None.",
+            "Not supported.",
+            "Not supported",
+            "Not available.",
+            "Not available",
+            "Not applicable",
+            "Not applicable."
+        };
+
+        private static string ConvertToThreeColumnPermissionsTable(IEnumerable<string> tableRows)
+        {
+            var tableString = "|Permission type|Least privileged permissions|Higher privileged permissions|" +
```

Comment:
Can we consider using a StringBuilder here instead of concatenating in the for loop below?